diff --git a/dist/index.js b/dist/index.js
index 7f343be..55ca90e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -457,7 +457,7 @@ function assignStyle(element, style) {
 }

 const TRANSITIONS = {
-    DURATION: 0.5,
+    DURATION: 0.15,
     EASE: [
         0.32,
         0.72,
diff --git a/dist/index.mjs b/dist/index.mjs
index 0dbbce5..553cdd1 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -435,7 +435,7 @@ function assignStyle(element, style) {
 }

 const TRANSITIONS = {
-    DURATION: 0.5,
+    DURATION: 0.15,
     EASE: [
         0.32,
         0.72,
